assets/js/modules/tagmanager/setup.js

```diff
@@ -51,7 +60,6 @@ class TagmanagerSetup extends Component {
 			containers: [],
 			errorCode: false,
 			errorMsg: '',
-			refetch: false,
 			selectedAccount: settings.accountID,
 			selectedContainer: settings[ containerKey ],
```

Comment:
See here. This could potentially make the value `undefined`. It never should for us because these values coming from PHP are strings, but still a bit error-prone.

I think we should get rid of the specific empty string value and the `UNSELECTED` constant again and just check `! value` instead.